src/App.js

```diff
@@ -1,10 +1,29 @@
 import { Route, Routes } from 'react-router-dom';
 import './App.css';
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import Books from './components/Books';
 import Categories from './components/Categories';
 import NavLinks from './components/NavLinks';
+import { getBookItems } from './redux/books/booksSlice';
 
 function App() {
+  const dispatch = useDispatch();
+
+  const { isLoading } = useSelector((state) => state.books);
+
+  useEffect(() => {
+    dispatch(getBookItems());
+  }, [dispatch]);
```

Comment:
- Kindly would you move the `useeffect` method to `booklist` component because you are fetching the data book list data when the app components start, that is what we don't want to be for your app performance and it is mentioned as part of the [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/connect_to_api.md) as a previous reviewer suggested👍